neptune/new/sync/__init__.py

```diff
@@ -115,42 +122,56 @@ def status(path: Path) -> None:
     metavar="project-name",
     help="project name (workspace/project) where offline runs will be sent",
 )
-def sync(path: Path, runs_names: List[str], project_name: Optional[str]):
-    """Synchronizes runs with unsent data with the server.
-
-    Neptune stores run data on disk in '.neptune' directories. In case a run executes offline
-    or network is unavailable as the run executes, run data can be synchronized
+def sync(
+    path: Path,
+    runs_names: List[str],
+    object_names: List[str],
+    project_name: Optional[str],
+):
+    """Synchronizes objects with unsent data with the server.
+
+    Neptune stores object data on disk in '.neptune' directories. In case a object executes offline
+    or network is unavailable as the run executes, object data can be synchronized
     with the server with this command line utility.
 
     You can list unsynchronized runs with `neptune status`
 
     Examples:
 
     \b
-    # Synchronize all runs in the current directory
+    # Synchronize all objects in the current directory
     neptune sync
 
     \b
-    # Synchronize all runs in the given path
+    # Synchronize all objects in the given path
     neptune sync --path foo/bar
 
     \b
     # Synchronize only runs "NPT-42" and "NPT-43" in "workspace/project" in the current directory
-    neptune sync --run workspace/project/NPT-42 --run workspace/project/NPT-43
+    neptune sync --object workspace/project/NPT-42 --object workspace/project/NPT-43
 
     \b
-    # Synchronise all runs in the current directory, sending offline runs to project "workspace/project"
+    # Synchronise all objects in the current directory, sending offline runs to project "workspace/project"
     neptune sync --project workspace/project
 
     \b
     # Synchronize only the offline run with UUID offline/a1561719-b425-4000-a65a-b5efb044d6bb
     # to project "workspace/project"
-    neptune sync --project workspace/project --run offline/a1561719-b425-4000-a65a-b5efb044d6bb
+    neptune sync --project workspace/project --object offline/a1561719-b425-4000-a65a-b5efb044d6bb
     """
 
     sync_runner = SyncRunner(backend=HostedNeptuneBackend(Credentials.from_token()))
 
     if runs_names:
-        sync_runner.sync_selected_runs(path, project_name, runs_names)
+        print("DAUPDUAPIDPOSDDSA")
+        click.echo(
+            "WARNING: --run parameter is deprecated and will be removed in the future, please start using --object"
+        )
+        # prefer object_names, obviously
+        if not object_names:
```

Comment:
I think there was a comment from Mycek, that we finally decided to accept both `--run` and `--object` parameters mixed, but here you accept only one of them.

Reply:
Oh god, hate that. But ok, I guess